solenoid_app/views.py

```diff
@@ -66,4 +50,80 @@ def formHandle(request):
             for k, v in data.items():
                 data[k] = str(v)
 
-            return JsonResponse(data, safe=False)
\ No newline at end of file
+            return JsonResponse(data, safe=False)
+
+def voltageChart(request):
+    labels = []
+    graph = []
+    data = {
+        'voltage': '',
+        'length': '',
+        'r_not': '',
+        'r_a': '',
+        'x': '',
+        'force': '',
+        'awg': '',
+        'compute': '',
+        'toGraph': ''
+    }
+
+    if (request.method == "POST"):
+        form = GraphForm(request.POST)
+        print(request.POST)
+        if form.is_valid():
+            data['voltage'] = form.cleaned_data['voltage']
+            data['length'] = form.cleaned_data['length']
+            data['r_not'] = form.cleaned_data['r_not']
+            data['r_a'] = form.cleaned_data['r_a']
+            data['x'] = form.cleaned_data['x']
+            data['force'] = form.cleaned_data['force']
+            data['awg'] = form.cleaned_data['awg']
+            data['compute'] = form.cleaned_data['compute']
+            data['toGraph'] = form.cleaned_data['toGraph']
+            compute = data['compute']
+
+            data['compute'] = None
+            data['force'] = None
+
+            if data['toGraph'] == 'voltage':
+                for volts in range(0, 51):
```

Comment:
I was talking to julian and with some playing around these values seem to work. This will likely change when we talk to Bart when we figure out what is best